pkg/operator/operator.go

```diff
@@ -482,3 +485,19 @@ func removeFinalizer(instance *v1alpha1.Driver, f string) {
 	}
 	instance.ObjectMeta.Finalizers = result
 }
+
+func (c *csiDriverOperator) recorderForInstance(instance *v1alpha1.Driver) events.Recorder {
+	// Guess Kind run-time, just in case it's changed in types.go
+	t := reflect.TypeOf(instance)
+	kind := t.Elem().Name()
+
+	ref := corev1.ObjectReference{
+		Kind:            kind,
+		Namespace:       instance.Namespace,
+		Name:            instance.Name,
+		UID:             instance.UID,
+		APIVersion:      v1alpha1.SchemeGroupVersion.String(),
+		ResourceVersion: instance.ResourceVersion,
+	}
+	return events.NewKubeRecorder(c.kubeClient.CoreV1().Events(""), operatorName, &ref)
```

Comment:
Is there any reason for using this instead of `NewRecorder()`?

Reply:
Not sure what's the difference between `NewRecorder` and `NewKubeRecorder`, but I got this error when using `NewRecorder`

```
an empty namespace may not be set during creation
```